core/dom/element.js

```diff
@@ -2113,6 +2113,16 @@ CKEDITOR.dom.element.clearMarkers = function( database, element, removeFromDatab
 				else if ( !type || node.type == type )
 					callback( node );
 			}
+		},
+
+		/**
+		 * Checks if element is not attached to DOM.
+		 *
+		 * @since 4.13.0
+		 * @returns {Boolean} Whether element is detached from DOM.
+		 */
+		isDetached: function() {
+			return !this.getDocument().getDocumentElement().contains( this );
```

Comment:
Is there a real use case to check if content of destroyed editor is detached?  If this comes from a listener - it will be removed with `editor.destroy()`.

Reply:
I think this will occur for some Edge cases in IE8 (and it's still better than the previous version) so maybe we can accept that fact? WDYT?